### DISCUSSION TITLE
Проблема с установкой плагина

### DISCUSSION THREAD

**Discussion by imcrazyhoudini** [PavelBlend/blender-xray#660] [Q&A]
Не могу поставить плагин.

Жму edit - preferences - add-ons - install - выбираю архив, пишет аддон установлен но по факту ничего не появляется.

Пробовал разные версии.

Блендер 3.6.

Есть решение какое?

**Comment by PavelBlend** [comment]
@imcrazyhoudini а какой именно архив ты выбираешь? Нужно скачать `blender-xray-2.31.1.zip` из раздела `Assets` отсюда:
https://github.com/PavelBlend/blender-xray/releases/tag/v2.31.1

Если ты скачивал с помощью зелёной кнопки на [главной странице](https://github.com/PavelBlend/blender-xray), то аддон не установится.

**Reply by imcrazyhoudini** [reply]
Оттуда и качал.

Решил проблему установкой папки 3.6 что у друга была:)
